src/components/panels/KlippyStatePanel.vue

```diff
@@ -1,105 +1,83 @@
-                    {{ $t('Panels.KlippyStatePanel.PrinterSwitchedOff') }}
-                </p>
-                <p>{{ $t('Panels.KlippyStatePanel.PrinterSwitchedOffDescription') }}</p>
+        <v-alert v-if="klippyIsConnected" :color="messageType.color" dense text border="left" class="mb-0 mb-6">
+            <!-- KLIPPER MESSAGE TITLE -->
+            <p class="font-weight-medium d-flex align-center">
+                <v-icon :color="messageType.color" class="pr-2">{{ messageType.icon }}</v-icon>
+                {{ serviceReportsKlipper }}
+            </p>
+            <!-- KLIPPER MESSAGE -->
+            <div v-if="klippy_message !== null">
+                <pre style="white-space: pre-wrap">{{ klippy_message.trim() }}</pre>
+                <v-divider class="mt-2 pb-3" />
                 <v-row>
-                    <v-col class="text-center">
-                        <v-btn small outlined text :class="`${messageType.color}--text my-1`" @click="powerOn">
-                            <v-icon class="mr-sm-2">{{ mdiPower }}</v-icon>
-                            {{ $t('Panels.KlippyStatePanel.PowerOn') }}
+                    <!-- RESTART BUTTONS -->
+                    <v-col>
+                        <v-btn small outlined text :class="buttonClasses" @click="restart">
+                            <v-icon class="mr-sm-2">{{ mdiRestart }}</v-icon>
+                            {{ $t('Panels.KlippyStatePanel.Restart') }}
+                        </v-btn>
+                        <v-btn small outlined text :class="buttonClasses" @click="firmwareRestart">
+                            <v-icon class="mr-sm-2">{{ mdiRestart }}</v-icon>
+                            {{ $t('Panels.KlippyStatePanel.FirmwareRestart') }}
+                        </v-btn>
+                    </v-col>
+                    <!-- LOG DOWNLOAD BUTTONS -->
+                    <v-col>
+                        <v-btn
+                            :href="apiUrl + '/server/files/klippy.log'"
+                            small
+                            outlined
+                            text
+                            :class="buttonClasses"
+                            @click="downloadLog">
+                            <v-icon class="mr-2">{{ mdiDownload }}</v-icon>
+                            {{ $t('Panels.KlippyStatePanel.KlipperLog') }}
+                        </v-btn>
+                        <v-btn
+                            :href="apiUrl + '/server/files/moonraker.log'"
+                            small
+                            outlined
+                            text
+                            :class="buttonClasses"
+                            @click="downloadLog">
+                            <v-icon class="mr-2">{{ mdiDownload }}</v-icon>
+                            {{ $t('Panels.KlippyStatePanel.MoonrakerLog') }}
                         </v-btn>
                     </v-col>
```

Comment:
you refactored the restart buttons to take less lines but not the download buttons.
Is there a special reason?